.github/workflows/go-windows.yml

```
@@ -14,7 +14,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        go-version: [1.17.x]
+        go-version: [1.18.x, 1.19.x]
```

```suggestion
        go-version: [1.17.x, 1.18.x, 1.19.x]
```
IMO we should keep 3 active versions for `minio-go`. For other libs (madmin, mc, minio) we can be more aggressive.

Implementing a `strings.Cut` isn't really a big deal.

---

I thought go1.17 is deprecated? @klauspost

---

Okay got back go1.17 for now.